### DISCUSSION TITLE
Resize image to window (Don't resize window on image change)

### DISCUSSION THREAD

**Discussion by SullyPanda76cl** [sylikc/jpegview#175] [Ideas]
Hi, i would gladly add this feature myself.

**Comment by sylikc** [comment]
Does `Zoom > Fit to Screen` work?

**Reply by jcclow** [reply]
This does not work. Wondering if anyone knows how to prevent the window from resizing when zooming. Sort of defeats the purpose of making the window small so you can have multiple windows on one screen and then trying to zoom in on an image.

**Comment by Samillion** [comment]
I mentioned in: https://github.com/sylikc/jpegview/issues/276#issuecomment-2099783845

I adjusted the following settings, which did that for me:
- `ShowFullScreen=false`
- `DefaultWindowRect=sticky`

The only issue is, if you maximize the window then close JPEGView, it kinda gets stuck like that until you manually resize the window to be smaller than "maximized".

Edit:
Maybe an improvement would be an option to control "unmaximize" value. (ie: 2/3 of screen size)
